SKILL.tsv

```diff
@@ -4797,175 +4797,175 @@ SKILL_20150717_004796	When an enemy gets shot by [Flash Canon], it will fall int
 SKILL_20150717_004797	Flesh Hoop: Demoralize
 SKILL_20150717_004798	Attak of enemies nearby the hoop after using [Flesh Hoop] decrease for 4 seconds. [Weakened ATK] can be overlapped max 10 times.
 SKILL_20150717_004799	The enemies near Shoggoth will fall into Decay status for 15 secs. This ability will last for 1 min per Attribute level after Shoggoth gets summoned.
-SKILL_20150717_004800	The range of [Disinter] will be increased by 10.
-SKILL_20150717_004801	Cooldown time of [Dig] will be reduced by 10 secs.
-SKILL_20150717_004802	Physical ATK and Magic ATK will be increased for 30 mins for the allies that participated in [Item Awakening] dungeon. ATK will increase by 20% per Attribute level.
-SKILL_20150717_004803	Fulldraw : Knockback increase
-SKILL_20150717_004804	[Both hands bow] to the public type monster when mounted I will give a 20% additional damage of Physical attack.
-SKILL_20150717_004805	When [Swift Steps] is being activated, the enemies that got shot by [Oblique Shot] will fall into [Slow] with 5% probability per Attribute level for 7 secs.
-SKILL_20150717_004806	Physical DEF of the enemies that are threaded with [Full Draw] will decrease by 10% per Attribute level.
+SKILL_20150717_004800	Increases the range of [Disinter] by 10.
+SKILL_20150717_004801	Reduces the cooldown of [Dig] by 10 secs.
+SKILL_20150717_004802	Physical ATK and Magic ATK will be increased for 30 mins for allies that participated in the [Item Awakening] dungeon. ATK will increase by 20% per Attribute level.
+SKILL_20150717_004803	Fulldraw : Increases knockback
```

Comment:
'Knockback Increase' is fine. 'Increases knockback' sounds odd. Maybe 'Knockback Reinforce'?